### PR TITLE
[F-25]: update meteor desktop

### PR DESCRIPTION
**Pull request by marozartsem9**
https://yt.sharekey.com/agiles/110-34/current?issue=F-25

What was done:
- merged branch with electron-update from (https://github.com/Meteor-Community-Packages/meteor-desktop/pull/8)
- fix issue with single instance: syntax was changed in the newer version of electron.

### CODE REVIEW

File: plugins/bundler/package.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable prefer-arrow-callback */
 Package.describe({
-    name: 'skadmin:meteor-desktop-bundler',
-    version: '2.3.0',
+    name: 'communitypackages:meteor-desktop-bundler',
```
Comment (Strangerxxx) [inline plugins/bundler/package.js:3]: this package must be under skadmin namespace

File: plugins/watcher/package.js [hunk 2]
```diff
@@ -1,22 +1,22 @@
 /* eslint-disable prefer-arrow-callback */
 Package.describe({
-    name: 'skadmin:meteor-desktop-watcher',
-    version: '2.3.0',
+    name: 'communitypackages:meteor-desktop-watcher',
```
Comment (Strangerxxx) [inline plugins/watcher/package.js:3]: this package must be under skadmin namespace

File: plugins/watcher/package.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-    chokidar: '2.0.2'
+    chokidar: '3.5.3'
 });
 
 Package.onUse(function onUse(api) {
     api.versionsFrom('METEOR@1.3.3');
     api.use('ecmascript');
     api.use([
-        'skadmin:meteor-desktop-bundler@2.3.0',
+        'communitypackages:meteor-desktop-bundler@3.0.0',
```
Comment (Strangerxxx) [inline plugins/watcher/package.js:19]: this package must be under skadmin namespace

File: plugins/watcher/watcher.js [hunk 4]
```diff
@@ -215,20 +215,20 @@ if (!('desktopHCP' in settings) || !settings.desktopHCP) {
     console.warn('[meteor-desktop] will not watch for changes is .desktop because there is no ' +
         '.desktop/settings.json or desktopHCP is set to false.  Remove this plugin if you do ' +
         'not want to use desktopHCP.');
-} else if ('skadmin:meteor-desktop-bundler' in Package) {
+} else if ('communitypackages:meteor-desktop-bundler' in Package) {
```
Comment (Strangerxxx) [inline plugins/watcher/watcher.js:218]: this package must be under skadmin namespace

File: plugins/watcher/watcher.js [hunk 5]
```diff
@@ -215,20 +215,20 @@ if (!('desktopHCP' in settings) || !settings.desktopHCP) {
...
     const chokidar = Npm.require('chokidar');
     const versionFile = path.join(rootPath, 'version.desktop');
 
     const version =
-        typeof Package['skadmin:meteor-desktop-bundler'].METEOR_DESKTOP_VERSION === 'object' ?
-            Package['skadmin:meteor-desktop-bundler'].METEOR_DESKTOP_VERSION.version : null;
+        typeof Package['communitypackages:meteor-desktop-bundler'].METEOR_DESKTOP_VERSION === 'object' ?
+            Package['communitypackages:meteor-desktop-bundler'].METEOR_DESKTOP_VERSION.version : null;
```
Comment (Strangerxxx) [inline plugins/watcher/watcher.js:224]: this package must be under skadmin namespace

File: plugins/watcher/watcher.js [hunk 6]
```diff
@@ -215,20 +215,20 @@ if (!('desktopHCP' in settings) || !settings.desktopHCP) {
...
 
     if (version) {
         try {
             fs.readFileSync(versionFile, 'UTF-8');
         } catch (e) {
             throw new Error('[meteor-desktop] there is no version.desktop file. Are you sure you ' +
-                'have skadmin:meteor-desktop-bundler package added to your project?');
+                'have communitypackages:meteor-desktop-bundler package added to your project?');
```
Comment (Strangerxxx) [inline plugins/watcher/watcher.js:231]: this package must be under skadmin namespace

File: plugins/watcher/watcher.js [hunk 7]
```diff
@@ -275,5 +275,5 @@ if (!('desktopHCP' in settings) || !settings.desktopHCP) {
     }
 } else {
     throw new Error('[meteor-desktop] bundler plugin was not detected. Are you sure you have ' +
-        'skadmin:meteor-desktop-bundler package added to your project?');
+        'communitypackages:meteor-desktop-bundler package added to your project?');
```
Comment (Strangerxxx) [inline plugins/watcher/watcher.js:278]: this package must be under skadmin namespace

File: tests/unit/meteorManager.test.js [hunk 8]
```diff
@@ -106,7 +106,7 @@ describe('meteorManager', () => {
             expect(
                 instance.checkPackagesVersion([
                     'omega:meteor-desktop-localstorage@0.0.11',
-                    'omega:meteor-desktop-watcher@0.0.11'
+                    'communitypackages:meteor-desktop-watcher@0.0.11'
```
Comment (Strangerxxx) [inline tests/unit/meteorManager.test.js:109]: this package must be under skadmin namespace

**Review by Strangerxxx — CHANGES_REQUESTED** [review]
as we have meteor-desktop hard forked we should have different namespace for meteor packages - skadmin

**Review by Strangerxxx — APPROVED** [review]
lgtm

**Comment by StorytellerCZ** [comment]
@marozartsem9 would you be willing to push your fixes upstream?

**Comment by marozartsem9** [comment]
> @marozartsem9 would you be willing to push your fixes upstream?

@StorytellerCZ , Hi!
https://github.com/Meteor-Community-Packages/meteor-desktop/pull/10

**Pull request merged**